### PR TITLE
Distmysql 186 release notes pdps8.0.28 update

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by george-lorch — APPROVED** [review]

File: source/release-notes-ps-v8.0.28.upd.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This update of |pdmysql| using |PS| is based on `Percona Server for MySQL 8.0.28-20 <https://www.percona.com/doc/percona-server/8.0/release-notes/Percona-Server-8.0.28-20.html>`_. It also includes the updated version of `Percona XtranBackup 8.0.28-21 <https://docs.percona.com/percona-xtrabackup/latest/release-notes/8.0/8.0.28-21.0.html>`_.
+
+Release Highlights
+=================================================
+
+* Encryption functions and variables to manage the encryption range are added. The functions may take an algorithm argument. Encryption converts plaintext into ciphertext using a key and an encryption algorithm.
+
+  You can also use the user-defined functions with the PEM format keys generated externally by the OpenSSL utility.
+
+* Support for the Amazon Key Management Service (KMS) component.
```
Comment (LenzGr) [inline source/release-notes-ps-v8.0.28.upd.rst]: Would it be possible to link to the documentation of this feature here?

File: source/release-notes-ps-v8.0.28.upd.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+:Installation: :ref:`installation`
+
+|pdmysql| is a single solution with the best and most critical enterprise components from the |mysql| open-source community, designed and tested to work together. 
+
+This update of |pdmysql| using |PS| is based on `Percona Server for MySQL 8.0.28-20 <https://www.percona.com/doc/percona-server/8.0/release-notes/Percona-Server-8.0.28-20.html>`_. It also includes the updated version of `Percona XtranBackup 8.0.28-21 <https://docs.percona.com/percona-xtrabackup/latest/release-notes/8.0/8.0.28-21.0.html>`_.
+
+Release Highlights
+=================================================
+
+* Encryption functions and variables to manage the encryption range are added. The functions may take an algorithm argument. Encryption converts plaintext into ciphertext using a key and an encryption algorithm.
```
Comment (LenzGr) [inline source/release-notes-ps-v8.0.28.upd.rst]: Would it be possible to link to the documentation of this feature here?

File: source/release-notes-ps-v8.0.28.upd.rst (hunk 1 above)
Comment (nastena1606) [reply source/release-notes-ps-v8.0.28.upd.rst]: Added

File: source/release-notes-ps-v8.0.28.upd.rst (hunk 2 above)
Comment (nastena1606) [reply source/release-notes-ps-v8.0.28.upd.rst]: Added

**Pull request merged**